src/Ant.cpp

```diff
@@ -2,26 +2,49 @@
 #include <antgame/Food.h>
 #include <antgame/World.h>
 #include <stdlib.h>
+#include <math.h>
 
 #include <easyloggingpp/easylogging++.h>
 
 #include <boost/geometry.hpp>
 
+
+#define PI 3.14159265
```

Comment:
get this from the math header somehow

Reply:
using boost solution from here for cross platform reasons https://stackoverflow.com/questions/1727881/how-to-use-the-pi-constant-in-c